src/content/docs/victory-histogram.md

```diff
@@ -0,0 +1,559 @@
+---
+id: 35
+title: VictoryHistogram
+category: charts
+type: docs
+scope:
+  - sampleHistogramData
+  - sampleHistogramDateData
+  - d3Scale
+  - d3Time
+  - d3Array
+---
+
+# VictoryHistogram
+
+VictoryHistogram renders a dataset as series of bars representing "bins", allowing you to see the distribution of your data. The data you pass in will be "binned" according to the `bin` prop you pass, allowing for flexibility in how these bins are determined. VictoryHistogram can be composed with [`VictoryChart`][] to create histogram charts.
+
+Because each bar represents a bin rather than a particular data point (like you would have with `VictoryScatter` for example), when accessing `datum` via a prop that passes `datum` such as `style`, datum will have properties `x`, `x0`, `x1`, `y`, and `binnedData`. `x` is the midpoint between the bin, `x0` is the beginning of the bin, `x1` is the end of the bin, `y` is the aggregate amount of data points within that bin, and `binnedData` is an array of the original data points that were grouped into this bin.
```

Comment:
This paragrah feels odd for the introduction. Maybe contrast use cases here with `VictoryBar` instead, and move the data detail to the data prop section

Reply:
Good idea, I left a note about using VictoryBar for qualitative data both in the data prop section and in the intro. I feel like it can't hurt to have a few mentions of it in case they miss it in one place.